packages/TesterApp/android/settings.gradle

```diff
@@ -1,12 +1,21 @@
-apply from: "./resolveNodePackage.gradle"
```

Comment:
I know that this is RePack's hack for PNPM to work with React Native apps, since we're right now migrating to RNTA - can we reuse work we can potentially reuse work from `rnx-kit` 🤔

Reply:
if you can make it work without this that's even better, not sure what else is here to discuss :D